backend/typescript/models/user.model.ts

```diff
@@ -1,9 +1,13 @@
 import { Column, DataType, Model, Table, HasOne } from "sequelize-typescript";
 import { Role } from "../types";
+import Volunteer from "./volunteer.model";
 import Donor from "./donor.model";
 
 @Table({ tableName: "users" })
 export default class User extends Model {
+  @HasOne(() => Volunteer, 'user_id')
```

Comment:
We actually do not need this, so we can remove this and also the import statement.